Add ignore: ["rules"] / ["at-rules"] to block-opening-brace-*-after #5262 (#5830)
stylelint/stylelint: lib/rules/block-opening-brace-newline-after/README.md

a {color: pink;
}
```

> # Optional secondary options
> 
> ### `ignore: ["rules"]`
> 
> Require a new line for at-rules, but allow single line for rules. This is usefull in single-line formatting.
> 
> For example :
> 
> The following patterns are _not_ considered problems:
> 
> <!-- prettier-ignore -->
> ```css
> a { color: pink; }
> ```
> 
> <!-- prettier-ignore -->
> ```css
> @media print {
>   a { color: pink; } 
> }
> ```

Reviewer: ```suggestion
## Optional secondary options

### `ignore: ["rules"]`

Ignore the opening brace of rules.

For example, with `"always"`:

The following pattern is _not_ considered a problem:

<!-- prettier-ignore -->
```css
a { color: pink; }
```
```
